docs/docs/README.md

```diff
@@ -3,25 +3,127 @@ prev: false
 
-If you're new to Pipedream, we recommend starting with this [5 minute tutorial](https://pipedream.com/new?tutorial=1) to learn how to:
+Watch a 4-minute demo or review our [quickstart guide](/quickstart/):
 
-- Trigger a workflow on HTTP requests
-- Retrieve data from an API using Node.js and npm modules
-- Deploy and test your changes
+<!--With Pipedream, you can stop writing boilerplate code, struggling with authentication and managing infrastructure, and start connecting APIs with code-level control when you need it — and no code when you don't. -->
 
-If you want to skip the tutorial, just [sign in](/sign-up/) to get started for free. If you have any questions or feedback, [join us on Slack](https://pipedream.com/community/) or [reach out](/support/).
+<!--Pipedream is a low code integration platform for developers. We make it easy to connect APIs remarkably fast so you can stop writing boilerplate code, struggling with authentication and managing infrastructure, and start connecting APIs with code-level control when you need it — and no code when you don't.-->
+
+<video controls poster="./images/demo-poster.png" width="100%">
+  <source src="https://res.cloudinary.com/pipedreamin/video/upload/v1612307285/homepage/Using_Event_Sources_and_Workflows__Analyze_Twitter_Sentiment_in_Real-Time_and_Save_to_Google_Sheets_ehy2ho.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+<!--![image-20210520194929461](./image-20210520194929461.png)-->
+
+<!--img src="https://res.cloudinary.com/pipedreamin/image/upload/v1612919959/homepage/workflow-demo_ks64up.png"-->
+
+<!--
+Trusted by developers from startups to Fortune 500 companies:
+
+![logos](https://res.cloudinary.com/pipedreamin/image/upload/v1612919944/homepage/logos_kcbviz.png)
+
+## How Pipedream Works
+
+Pipedream provides a serverless platform to build and run workflows that connect APIs:
+
+- Connect your OAuth and key-based API accounts in seconds
+- Use connected accounts to auth APIs in code steps or in "no code" building blocks
+- Compose steps into workflows and trigger on HTTP requests, schedules or app events
+
+Pipedream also provides easy to use services to solve common serverless and integration challenges including state management, execution rate and concurrency controls, large file support (up to 5TB) and more! 
+
+Watch a demo (4 mins):
+
+-->
+
+## Is Pipedream for me?
+
+We make it easy to connect APIs with code-level control when you need it — and no code when you don't. If you and your team want to stop writing boilerplate code, struggling with authentication and managing infrastructure for API integrations, then Pipedream is for you. 
+
+Developers with a working knowledge of Node.js or Javascript will get the most value from Pipedream (Python, Typescript. and GitHub integration are coming soon).
```

Comment:
Comma after TypeScript instead of period

Reply:
And it's technically it's "TypeScript" (camel-case)

Reply:
done